### ISSUE TITLE
Store all pages as you browse?

### ISSUE THREAD

**Issue by benwhalley** [oto-labs/librarian#8]
Would it be possible to store a text-representation of all pages as you browse, and search against that? Probably useful to be able to blacklist some domains, but would be super useful.

**Comment by kashyapakshay** [comment]
Yes, it should be possible. I think we just need to first address the scraping and smarter text subselection pieces first so we don't blow up local storage usage.
